deploy/crd.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   name: perconaxtradbclusters.pxc.percona.com
 spec:
+  preserveUnknownFields: false
```

Comment:
What are we solving with this?

Reply:
@egegunes please add task number to the commit message